### PR TITLE
[AzureRM] Bump Version

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: tools/VersionController/Models/VersionBumper.cs [hunk 1]
```diff
@@ -160,7 +160,9 @@ private void UpdateRollupModuleManifest()
             var file = File.ReadAllLines(rollupModuleManifestPath);
             var pattern = @"ModuleName(\s*)=(\s*)(['\""])" + moduleName + @"(['\""])(\s*);(\s*)RequiredVersion(\s*)=(\s*)(['\""])" + _oldVersion + @"(['\""])";
             var updatedFile = file.Select(l => Regex.Replace(l, pattern, "ModuleName = '" + moduleName + "'; RequiredVersion = '" + _newVersion + "'"));
-            File.WriteAllLines(rollupModuleManifestPath, updatedFile);
+            var pattern2 = @"ModuleName(\s*)=(\s*)(['\""])" + moduleName + @"(['\""])(\s*);(\s*)ModuleVersion(\s*)=(\s*)(['\""])" + _oldVersion + @"(['\""])";
+            var updatedFile2 = updatedFile.Select(l => Regex.Replace(l, pattern2, "ModuleName = '" + moduleName + "'; ModuleVersion = '" + _newVersion + "'"));
+            File.WriteAllLines(rollupModuleManifestPath, updatedFile2);
```
Comment (wyunchi-ms) [inline tools/VersionController/Models/VersionBumper.cs:165]:
```C#
var patternReplaceRequiredVersion = @"ModuleName(\s*)=(\s*)(['\""])" + moduleName + @"(['\""])(\s*);(\s*)RequiredVersion(\s*)=(\s*)(['\""])" + _oldVersion + @"(['\""])";
var patternReplaceModuleVersion = @"ModuleName(\s*)=(\s*)(['\""])" + moduleName + @"(['\""])(\s*);(\s*)ModuleVersion(\s*)=(\s*)(['\""])" + _oldVersion + @"(['\""])";
var updatedFile = file.Select(l => Regex.Replace(l, patternReplaceRequiredVersion, $"ModuleName = '{moduleName}'; RequiredVersion = '{_newVersion}'"))
                      .Select(l => Regex.Replace(l, patternReplaceModuleVersion, $"ModuleName = '{moduleName}'; ModuleVersion = '{_newVersion}'"));
File.WriteAllLines(rollupModuleManifestPath, updatedFile);
```
Maybe better?

Comment (BethanyZhou) [reply tools/VersionController/Models/VersionBumper.cs]: Yes, indeed. Thanks for your suggestion.

File: src/ResourceManager/AnalysisServices/Commands.AnalysisServices/AzureRM.AnalysisServices.psd1 [hunk 2]
```diff
@@ -116,20 +116,26 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = ' * Fixed issue with default resource groups not being set.
- * Updated common runtime assemblies'
+        ReleaseNotes = '* Microsoft Azure PowerShell - AnalysisServices service cmdlets.
```
Comment (dingmeng-xue) [inline src/ResourceManager/AnalysisServices/Commands.AnalysisServices/AzureRM.AnalysisServices.psd1:119]: It should not lead with asterisk except this line is a bullet.

File: src/ResourceManager/AnalysisServices/Commands.AnalysisServices/AzureRM.AnalysisServices.psd1 [hunk 3]
```diff
@@ -116,20 +116,26 @@ PrivateData = @{
...
+* This module is outdated and will go out of support on 29 February 2024.
+* The Az.AnalysisServices module has all the capabilities of AzureRM.AnalysisServices and provides the following improvements:
+    - Az.AnalysisServices takes advantage of greater security with token cache encryption and improved authentication.
+    - Availability in Azure Cloud Shell and on Linux and macOS.
+* We encourage you to start using the Az module as soon as possible to take advantage of these improvements.
+* [Update your scripts](https://aka.ms/azpsmigrate)?that use AzureRM PowerShell modules to use Az PowerShell modules by 29 February 2024.
+* To automatically update your scripts, follow the?[quickstart guide](https://aka.ms/azpsmigratequick).'
```
Comment (dingmeng-xue) [inline src/ResourceManager/AnalysisServices/Commands.AnalysisServices/AzureRM.AnalysisServices.psd1:126]: Why is there a question mark?
```suggestion
* To automatically update your scripts, follow the [quickstart guide](https://aka.ms/azpsmigratequick).'
```
```suggestion
* To automatically update your scripts, follow the?[quickstart guide](https://aka.ms/azpsmigratequick).'
```

File: tools/AzureRM/AzureRM.psd1 [hunk 4]
```diff
@@ -51,61 +51,61 @@ CLRVersion = '4.0'
...
-               @{ModuleName = 'AzureRM.AnalysisServices'; RequiredVersion = '0.6.14'; }, 
+               @{ModuleName = 'AzureRM.AnalysisServices'; RequiredVersion = '0.6.15'; }, 
                @{ModuleName = 'Azure.AnalysisServices'; RequiredVersion = '0.5.4'; }, 
-               @{ModuleName = 'AzureRM.ApiManagement'; RequiredVersion = '6.1.7'; }, 
-               @{ModuleName = 'AzureRM.ApplicationInsights'; RequiredVersion = '0.1.8'; }, 
-               @{ModuleName = 'AzureRM.Automation'; RequiredVersion = '6.1.1'; }, 
+               @{ModuleName = 'AzureRM.ApiManagement'; RequiredVersion = '6.1.8'; }, 
+               @{ModuleName = 'AzureRM.ApplicationInsights'; RequiredVersion = '0.1.9'; }, 
+               @{ModuleName = 'AzureRM.Automation'; RequiredVersion = '6.1.2'; }, 
                @{ModuleName = 'AzureRM.Backup'; RequiredVersion = '4.0.11'; },
```
Comment (dingmeng-xue) [inline tools/AzureRM/AzureRM.psd1:61]: Remove retired module from the list.

**Comment by dingmeng-xue** [comment]
Changelog under the root folder should be updated as well. Its content should be the same as release note in azurerm.psd1.

**Review by dingmeng-xue — APPROVED** [review]

**Pull request merged**